src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -1746,6 +1747,7 @@ def load_arguments(self, _):
         c.argument('status_code_ranges', help='Space-separated list of status codes in MIN-MAX or VAL format.', nargs='+', validator=validate_status_code_ranges)
 
     with self.argument_context('network traffic-manager profile update') as c:
+        c.argument('max_return', help="Maximum number of endpoints to be returned for MultiValue routing type.", type=int)
```

Comment:
As parameter 'max_return' is already registered in 'network traffic-manager profile' argument context, it can be omitted here.

Reply:
got it. I will delete the surplus code